packages/experience/src/Providers/CaptchaContextProvider/utiles.ts

```diff
@@ -0,0 +1,15 @@
+import { CaptchaType } from '@logto/schemas';
+
+import { type SignInExperienceResponse } from '@/types';
+
+export const getScript = (config: SignInExperienceResponse['captchaConfig']) => {
+  if (!config) {
+    throw new Error('Captcha config is not found');
```

Comment:
nit: add a comment this should not happen, otherwise we need I18n for the error message.